state_solver/suspension.py

```diff
@@ -13,43 +13,53 @@
 
 class Suspension():
     """ Handles suspension vehicle forces """
-    def __init__(self, params:vehicle_params.BaseVehicle, logger:engine.Logger):
+    def __init__(self, params, logger:Logger):
         self.logger = logger
-        self.params:vehicle_params.BaseVehicle = params
-        self.__tires = engine.Tires(params)
+        self.params = params
+        self.__tires = Tires(params)
 
 
-    def get_loads(self, vehicle_velocity:np.array, yaw_rate:float, steered_angle:float, roll:float, pitch:float, heave:float):
-        forces, moments = np.array([0, 0, 0]), np.array([0, 0, 0])
+    def get_loads(self, vehicle_velocity:np.array, yaw_rate:float, steered_angle:float, roll:float, pitch:float, heave:float, slip_ratios:list):
+        veh_forces, veh_moments, wheel_speeds, wheel_torques = np.array([0, 0, 0]), np.array([0, 0, 0]), np.array([]), np.array([])
```

Comment:
veh is a disgusting acronym,  change my mind 😮‍💨

Reply:
Waiting for your transformations PR 🥺